### ISSUE TITLE
Add annotation functionality

### ISSUE THREAD

**Issue by AndreaSuckro** [shoeffner/cvloop#13]
It would be nice, if it would be possible to add annotations to an image and visualize them similar to here: [example stackoverflow](http://stackoverflow.com/questions/37435369/matplotlib-how-to-draw-a-rectangle-on-image). Not sure what the best abstraction for this use-case would be though ✌️

**Comment by shoeffner** [comment]
This sounds like fun. I will look into it soon!

**Comment by AndreaSuckro** [comment]
Features interesting would be: one or more annotation points per frame and size, linewidth and color of the rectangle.

**Comment by shoeffner** [comment]
I have implemented an annotation feature in #16 . Do you mind checking it out?

You can find some examples in the [examples notebook](https://github.com/shoeffner/cvloop/blob/feature/annotations/examples/cvloop_examples.ipynb), but basically it boils down to:

```
from cvloop import cvloop

annotations = []
for frame in range(20, 51):
    annotations.append([300, 250, frame, {
        'shape': 'RECT',
        'color': (0.3, 0.2, 0.8),
        'line': 4,
        'size': (30, 80)
    }])

cvloop('768x576.avi', annotations=annotations)
```

**Comment by shoeffner** [comment]
Closed with #16 and #17.
